x-pack/plugins/infra/public/plugin.ts

```diff
@@ -103,6 +103,7 @@ export class Plugin implements InfraClientPluginClass {
                     entries: [
                       { label: 'Inventory', app: 'metrics', path: '/inventory' },
                       { label: 'Metrics Explorer', app: 'metrics', path: '/explorer' },
+                      { label: 'Hosts', app: 'metrics', path: '/hosts' },
```

Comment:
I'm curious about the order of links here, I'd say hosts should be at least above explorer, if not above inventory (but that might change over time as inventory becomes less about hardware and more about "supporting services")

Reply:
It is supposed to be the first link according to the mockups, I just stuck it under here because due to its unfinished state and the fact we want to have it behind a technical preview, it seemed least important, but I don't think that's how we're ordering, so I'll move just move it to the top.